tls/s2n_security_policies.c

```diff
@@ -951,8 +951,7 @@ int s2n_security_policies_init()
             }
 
             /* Sanity check that valid tls13 has minimum tls version set correctly */
-            S2N_ERROR_IF(s2n_is_valid_tls13_cipher(iana) ^
-                (cipher->minimum_required_tls_version >= S2N_TLS13), S2N_ERR_INVALID_SECURITY_POLICY);
+            S2N_ERROR_IF(s2n_is_valid_tls13_cipher(iana) ^ (cipher->minimum_required_tls_version >= S2N_TLS13), S2N_ERR_INVALID_SECURITY_POLICY);
```

Comment:
formatting looks odd. wondering if its one of the changes you plan to address later.

Reply:
This can easily be fixed manually. Will do that.